Sources/Foundation/Data.swift

```diff
@@ -36,6 +36,7 @@
 @usableFromInline let memcpy = Musl.memcpy
 @usableFromInline let memcmp = Musl.memcmp
 #elseif canImport(WASILibc)
+@_implementationOnly import wasi_emulated_mman
```

Comment:
@kateinoigakukun WDYT of using `private import` here instead? `@_implementationOnly` should not be used for non-resilient libraries, so maybe `private import` is what we want? You may need to pass `-enable-experimental-feature AccessLevelOnImport` for that to work.

Reply:
For `wasi_emulated_mman` module, both `private import` and `@_implementationOnly` are ok because `wasi_emulated_mman` module is a publicly shipped module. For `CoreFoundation` module, we can't use the current `private import` implementation because `private import A` in module B requires module A to be found when module B is imported by other modules, and CoreFoundation is going to be removed from the toolchain distribution.

Reply:
I prefer we use `wasi_emulated_mman` here if possible.